fearless/ApplicationLayer/Services/ScamService/ScamService.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+import RobinHood
+
+protocol ScamServiceOperationFactoryProtocol {
+    func verifyOperation(for address: String) -> BaseOperation<ScamInfo?>
```

Comment:
verify operation should return "Bool" type. if you want to fetch ScamInfo by address, name it fetchOperation please